src/components/filter/checkboxController.ts

```diff
@@ -0,0 +1,33 @@
+import { IChooseParams } from "./chooseParamsObj";
+import { Params } from "./drawFilters/index";
+
+export function getInputValue(
+  event: Event,
+  param: string,
+  filterParams: IChooseParams
+) {
+  const ev = event.target as HTMLElement;
```

Comment:
it is worth choosing more meaningful names for variables, this will make the code more readable and cleaner